packages/react/src/components/Slider/useSlider.ts

```diff
@@ -86,9 +97,18 @@ export const useSlider = (props: ISliderProps, ref: React.Ref<HTMLDivElement>) =
     props.defaultValue,
     (ev: React.FormEvent<HTMLElement> | undefined, v: ISliderProps['value']) => props.onChange && props.onChange(v!),
   );
+  const [unclampedLowerValue, setLowerValue] = useControllableValue(
+    props.value,
+    props.min,
+    (ev: React.FormEvent<HTMLElement> | undefined, v: ISliderProps['value']) => props.onChange && props.onChange(v!),
+  );
+
+  const shouldChangeLowerValueRef = React.useRef<boolean>(false);
```

Comment:
I agree that the name is not ideal here. `isLowerThumbFocusedRef` is better but still might not be the best for precision. Because the ref here is used to determine which thumb value to update for both keyboard and mouse events. The events could be triggered not only by focusing on the thumb, but also clicking on the sliderBox. Maybe `shouldMoveLowerThumb` or `shouldUpdateLowerThumb`? Or do you have any other ideas?